---
title: BioNetVisA workshop 2018
layout: default
---

<center><a href="/"><img id="ECCB18" src="/Logo_ECCB2018.gif" style="width:160px;"/></a></center>
<p> </p>

# <center><font color="#e60000">BioNetVisA 2018 workshop</font></center>
## <center><font color="#e60000">From biological network reconstruction to data visualization and analysis in molecular biology and medicine</font></center>
<p align="center"><b><a href="https://easychair.org/conferences/?conf=bionetvisa2018">Extended abstract submission deadline: August 10!</a></b></p>

<p align="justify"><b>BioNetVisA</b> workshop brings together different actors of network biology from database providers, networks creators, computational biologists, biotech companies involved in data analysis and modeling to experimental biologists, clinicians that use systems biology approaches. The participants are exposed to the different paradigms of network biology and the latest achievements in the field.
</p>

### Motivation

<p align="justify">The goal of <b>BioNetVisA</b> workshop is to build a discussion around various approaches for biological knowledge formalisation, data integration and analysis; compatibility between different methods and biological networks resources available the field; applicability for concrete research and clinical projects depending on scientific question and type of high-throughput data.
</p>

<p align="justify">The <b>BioNetVisA</b> workshop aims at identifying bottlenecks and proposing short- and long-term objectives for the community as discussing questions about accessibility of available tools for wide range of user in every-day standalone application in biological and clinical labs. In addition, the possibilities for collective efforts by academic researchers, clinicians, biotech companies and future development directions in the field will be discussed during the round table panel.
</p>

### BioNetVisA 2018 programme

<table>
<tr><td><strong><font color="#e60000">09:00 - 10:30 Session 1: Signalling and metabolic network resources</font></strong></td></tr>
<tr><td><b>09:00 - 09:20 <a href="http://www.sbi.jp/members.htm">Samik Ghosh</a></b><br />
<i>The Systems Biology Institute, Tokyo, Japan</i><br />
<strong>Towards a network-centric approach in systems toxicology</strong><br />
<a href="/Abstracts/Bionetvisa2018/Samik_Ghosh_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>09:20 - 09:35 <a href="https://sysbio.curie.fr/">Cristobal Monraz Gomez</a></b><br />
<i>Institut Curie, Paris, France</i><br />
<strong>Atlas of Cancer Signaling Network: a resource of multi-scale biological maps to study disease mechanisms</strong><br />
<a href="/Abstracts/Bionetvisa2018/Cristobal_Monraz_Gomez_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><b>09:35 - 09:55 <a href="http://peoplewiki.clinbioinfosspa.es/jdopazo/">Joaquin Dopazo</a></b><br />
<i>Fundación Progreso y Salud, Seville, Spain</i><br />
<strong>Navigating through disease maps</strong><br />
<a href="/Abstracts/Bionetvisa2018/Joaquin_Dopazo_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><b>09:55 - 10:10 <a href="https://www.ebi.ac.uk/about/people/antonio-fabregat-mundo">Antonio Fabregat</a></b><br />
<i>EMBL-EBI, Hinxton, UK</i><br />
<strong>The Reactome Pathway Knowledgebase</strong><br />
<a href="/Abstracts/Bionetvisa2018/Antonio_Fabregat_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><b>10:10 - 10:30 <a href="https://lunean.com/"><mark>Augustin Luna</mark></a></b><br />
<i><mark>Dana-Farber Cancer Institute/Harvard University, Boston, United States</mark></i><br />
<strong><mark>TBA</mark></strong></td></tr>
</table>

<table>
 <tr><td><i>10:30 - 11:00 Coffee break</i></td></tr>
</table>

<table>
<tr><td><strong><font color="#e60000">11:00 - 12:30 Session 2: Platforms and methods for analysis of complex networks</font></strong></td></tr>
<tr><td><b>11:00 - 11:20 <a href="http://pavlopouloslab.info">Georgios Pavlopoulos</a></b><br />
<i>Biomedical Sciences Research Center "Alexander Fleming", Athens, Greece</i><br />
<strong>Using HipMCL, a high-performance parallel implementation of the Markov clustering algorithm, to understand microbial diversity</strong><br />
<a href="/Abstracts/Bionetvisa2018/Georgios_Pavlopoulos_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>11:20 - 11:40 <a href="https://www.researchgate.net/profile/Anna_Niarakis">Anna Niarakis</a></b><br />
<i>GenHotel EA3886, Univ Evry, Université Paris-Saclay, France</i><br />
<strong>Disease Networks - Reconstruction, Topology, Dynamics. Towards an automated pipeline from static representations to executable disease models</strong><br />
<a href="/Abstracts/Bionetvisa2018/Anna_Niarakis_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>11:40 - 11: 55 <a href="https://www.researchgate.net/profile/Josephine_Daub">Josephine Daub</a></b><br />
<i>Princess Máxima Center for Pediatric Oncology, Utrecht, The Netherlands</i><br />
<strong>Genetic interactions in childhood cancer</strong><br />
<a href="/Abstracts/Bionetvisa2018/Josephine_Daub_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>11:55 - 12: 10 <a href="https://www.research.ed.ac.uk/portal/en/persons/barbara-shih(0a863546-b83e-4884-90cc-4d796da614d4).html">Barbara Shih</a></b><br />
<i>The Roslin Institute and Royal (Dick) School of Veterinary Studies, the University of Edinburgh, Easter Bush, Midlothian, Edinburgh, UK</i><br />
<strong>A platform for the network assembly and visual analysis of transcript isoforms from short-read RNA-sequencing data</strong><br />
<a href="/Abstracts/Bionetvisa2018/Barbara_Shih_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>12:10 - 12:30 Talks selected from abstracts</b></td></tr>
</table>

<table>
<tr><td><i>12:30 - 13:30 Lunch</i></td></tr>
</table>

<table>
<tr><th><font color="#e60000">13:30 - 15:00 Session 3: Biological networks in single-cell data analysis</font></th></tr>
<tr><td><b>13:30 - 13:50 <a href="http://www.vib.be/en/research/scientists/Pages/Yvan-Saeys-Lab.aspx">Yvan Saeys</a></b><br />
<i>VIB-Ghent University, Gent, Belgium</i><br />
<strong>Inferring dynamic regulatory networks from single cell data</strong><br />
<a href="/Abstracts/Bionetvisa2018/Yvan_Sayes_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>13:50 - 14:10 <a href="https://sysbio.curie.fr/people.html">Andrei Zinovyev</a></b><br />
<i>Institute Curie, Paris, France</i><br />
<strong>Biological networks help unraveling tumoral heterogeneity at single cell level</strong><br /></td></tr> 
<tr><td><b>14:10 - 14:25 <a href="https://www.researchgate.net/profile/Anirudh_Patir3">Anirudh Patir</a></b><br />
<i>The University of Edinburgh, Edinburgh, UK</i><br />
<strong>Evaluation of network methods for the analysis scRNA-seq data and development of a new KNN-based method based on identified caveats</strong><br />
<a href="/Abstracts/Bionetvisa2018/Anirudh_Patir_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>14:25 - 15:00 Talks selected from Abstracts</b></td></tr>
</table>

<table>
<tr><td><i>15:00 - 15:30 Coffee break</i></td></tr>
</table>

<table>
<tr><th><font color="#e60000">15:30 - 17:00 Session 4: Biological network in drug repositioning and disease comorbidity</font></th></tr>
<tr><td><b>15:30 - 16:10 <a href="https://www.bsc.es/valencia-alfonso">Alfonso Valencia</a> Keynote Lecturer</b><br />
<i>Barcelona Supercomputing Center, Barcelona, Spain</i><br />
<strong>Disease networks for the study of the molecular basis of disease comorbidity</strong></td></tr>
<tr><td><b>16:10 - 16:25 <a href="http://www.ibens.ens.fr/spip.php?article95">Celine Hernandez</a></b><br />
<i>Institut de Biologie de l’Ecole Normale Supérieure, Paris, France</i><br />
<strong>Dynamical modelling of T cell co-inhibitory pathways to predict anti-tumour responses to checkpoint inhibitors</strong><br />
<a href="/Abstracts/Bionetvisa2018/Celine_Hernandez_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>16:25 - 16:40 <a href="https://www2.unil.ch/cbg/index.php?title=Sven_Bergmann">Sven Bergmann</a></b><br />
<i>University of Lausanne & Swiss Institute of Bioinformatics, Switzerland</i><br />
<strong>Open Community Challenge Reveals Molecular Network Modules with Key Roles in Diseases</strong><br />
<a href="/Abstracts/Bionetvisa2018/Sven_Bergmann_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>16:40 - 17:00 Round table discussion</b></td></tr>
</table>

### Topics
<p>Graphical representation of biological knowledge<br />
Pathway and network resources<br />
Networks annotation and curation<br />
Multi-omics data visualization and analysis in the context of networks<br />
Network modelling<br />
Basic research and clinical application of networks<br />
Microbiome and networks<br />
Networks for single-cell data interpretation<br />
Metabolomics and networks<br />
Networks applications for drug repositioning<br />
Networks in disease comorbidity studies<br />
</p>

### Scope
<p align="justify"><b>Development, curation and maintenance of biological network resources</b><br />
Data models from interaction to pathway and network, available tools for the integration of new discoveries into existing signalling diagrams. Discussion of curation models from map manager-oriented to community-based update using web-tools and forums.
</p>

<p align="justify"><b>Cell signalling knowledge representation paradigms</b><br />
From interaction to single pathways representation to comprehensive signalling networks: discussion of modes of biological processes representation; granularity, coverage, sources of information, annotation modes, cross-citations between different signalling networks resources, etc. Standard for graphical representation and data formats.
</p>

<p align="justify"><b>Visualization of data through networks</b><br />
Paradigms of visualization of different types of high-throughput data on a network map. Data integration with a map. Methods compatibility with different paradigms of signalling networks reconstruction in various databases and resources.
</p>

<p align="justify"><b>Analytical tools for network biology</b><br />
Tools for network analysis and manipulation, network modeling, network-based statistics. Studying network activation and inhibition, feedbacks and compensations, synthetic interactions.
</p>

<p align="justify"><b>Applications of networks in multi-omics data analysis in biomedical research and in clinics</b><br />
Studies on single-cell data, microbiome, metabolomics and other omics data analysis, disease
comorbidity, drug response prediction using network-based approaches.
</p>


### Abstract submission instructions

We invite to submit your abstract for <b>Talk and/or poster</b> in the topics described above.

#### <font color="#e60000">The submission deadline is August 10th 2018</font>

<p>Submitted abstracts will be reviewed by the scientific program committee and notification of acceptance for <b>talk and/or poster</b> will be provided to the corresponding author by August 12th, 2018.
</p>

#### <font color="#e60000">Late abstract submission for posters only is possible till August 9th  2018</font>

<p>Number of poster abstracts will be selected for flash poster presentations (3 minutes for topic advertised).</p>

<p><b>Abstract format:</b><br />
Title<br />
List of authors (the first author is the presenting author)<br />
Affiliations<br />
Abstract text in unstructured format (maximum 300 words)
</p>
  
<p align="justify"><b>Submission process</b><br />
The link below will take you to the EasyChair submission page. Once logged, click on the 'Submission' tab to start the submission process. Enter authors, title without HTML elements, abstract up to 300 words, keywords, and select up to <b><span style="text-decoration: underline;">three</span></b> relevant topic sessions.
</p>

<p align="justify">To assist us with creating the abstract booklet, please upload the same abstract as a word document in the section called "Upload Paper" using this <b><a href="/AbstractTemplate_BioNetVisA.docx"><span style="text-decoration: underline;"><font color="#e60000">TEMPLATE</font></span></a></b>
</p>

<p align="justify">When the form is adequately filled out, press the 'Submit' button. You will receive an email from the conference "EasyChair". This email is solely a notification that EasyChair for BioNetVisA2014 has received the abstract.
</p>

<b><a href="https://easychair.org/conferences/?conf=bionetvisa2018"><font size="5" color="#e60000"><center>Submit your abstract here</center></font></a></b>

### Registration

<p>The registration has to be performed via the <b><a href="http://eccb18.org/registration/"><font color="#e60000">ECCB website</font></a></b>
</p>

### Organising committee
<p><a href="http://sysbio.curie.fr/">Inna Kuperstein</a> (Institut Curie, France)<br />
<a href="https://science.curie.fr/recherche/biologie-interactive-des-tumeurs-immunologie-environnement/c/">Emmanuel Barillot</a> (Institut Curie, France)<br />
<a href="http://www.ihes.fr/~zinovyev/">Andrei Zinovyev</a> (Institut Curie, France)<br />
<a href="http://sysbio.curie.fr/">Luis Cristobal Monraz Gomez</a> (Institut Curie, France)<br />
<a href="http://www.sbi.jp/">Hioraki Kitano</a> (RIKEN Center for Integrative Medical Sciences, Japan)<br />
<a href="http://www.kanehisa.jp/en/kanehisa.html">Minoru Kanehisa</a> (Institute for Chemical Research, Kyoto University, Japan)<br />
<a href="http://www.sbi.jp/members.htm">Samik Ghosh</a> (Systems Biology Institute, Tokyo, Japan)<br />
<a href="https://lenoverelab.org/">Nicolas Le Novère</a> (Babraham Institute, UK)<br />
<a href="https://reactome.org/">Robin Haw</a> (Ontario Institute for Cancer Research, Canada)<br />
<a href="https://www.bsc.es/valencia-alfonso">Alfonso Valencia</a> (Spanish National Bioinformatics Institute, Madrid, Spain)<br/>
<a href="https://www.nki.nl/divisions/molecular-carcinogenesis/wessels-l-group/">Lodewyk Wessels</a> (Netherlands Cancer Institute, Amsterdam, Netherlands)<br />
  <a href="https://www.prinsesmaximacentrum.com/">Patrick Kemmeren</a> (Princess Maxima Center for Pediatric Oncology, Utrecht, Netherlands)
</p>

### Sponsors
TBA
