---
title: BioNetVisA workshop 2019
---

<center><a href="/"><img id="ICSB19" src="/ISCB2019_logo.jpg" style="width:160px;"/></a></center>
<p> </p>

# <center><font color="#e60000">BioNetVisA 2019 workshop</font></center>
## <center><font color="#e60000">From biological network reconstruction to data visualization and analysis in molecular biology and medicine</font></center>
<!--
<p align="center"><b><font color="#e60000">The final programme and the abstract booklet are available</font><a href="/Abstracts/Bionetvisa2018/BioNetVisA2018_booklet.pdf"> here</a></b></p>
<p align="center"><b><font color="#e60000">The workshop presentations are available</font><a href="https://drive.google.com/open?id=1cLa4gSUAeogCNUi1yAFzlidHbOrxakxh"> here</a></b></p> -->

<p align="justify"><b>BioNetVisA</b> workshop brings together different actors of network biology from database providers, networks creators, computational biologists, biotech companies involved in data analysis and modeling to experimental biologists, clinicians that use systems biology approaches. The participants are exposed to the different paradigms of network biology and the latest achievements in the field.
</p>

### Motivation

<p align="justify">The goal of <b>BioNetVisA</b> workshop is to build a discussion around various approaches for biological knowledge formalisation, data integration and analysis; compatibility between different methods and biological networks resources available the field; applicability for concrete research and clinical projects depending on scientific question and type of high-throughput data.
</p>

<p align="justify">The <b>BioNetVisA</b> workshop aims at identifying bottlenecks and proposing short- and long-term objectives for the community as discussing questions about accessibility of available tools for wide range of user in every-day standalone application in biological and clinical labs. In addition, the possibilities for collective efforts by academic researchers, clinicians, biotech companies and future development directions in the field will be discussed during the round table panel.
</p>

### Audience
<p align="justify">The workshop targets computational systems biologists, molecular and cell biologists, clinicians and a wide audience interested in update and discussion around current status of network biology, pathway databases, and related analysis tools, including visualization, statistical analysis and dynamic modelling. No computational background is required to attend the workshop.
</p>

### Key Themes
<p>Graphical representation of biological knowledge<br />
Molecular interaction and pathway databases<br />
Comprehensive signalling networks<br />
Networks annotation and curation<br />
High-throughput data visualization, analysis and interpretation in the context of networks<br />
Multi-scale networks (genome, epigenome, transcriptome, proteome, metabolome...)<br />
Network modelling<br />
Machine learning/Artificial Intelligence approaches in network biology <br />
Basic research and clinical application of networks<br />
Microbiome and networks<br />
Single-cell data and network inference<br />
Metabolomics and networks<br />
Human microbiome<br />
Networks for drug repositioning<br />
</p>

### Important dates
<table style="width 100%">
<tr><td><b>August 1</b></td>
  <td>Abstract submission deadline<b><i><font color="#e60000">Submit now!</font></i></b></td></tr>
<tr><td><b>August 7</b></td>
  <td>Abstract acceptance notification</td></tr>
<tr><td><b>August 20</b></td>
  <td>Workshop's final program announcements</td></tr>
<tr><td><b>September 1</b></td>
  <td>Workshop's registration deadline (no fee)</td></tr>
<tr><td><b>October 31</b></td>
  <td>BioNetVisA 2019</td></tr>
</table>

### Preliminary program

<table>
<tr><td><strong><font color="#e60000">Session 1: Pathway resources and maps construction in the AI era</font></strong></td></tr>
<tr><td><b>08:50 - 09:00 Opening</b></td></tr>
<tr><td><b>09:00 - 09:25</b><br />
<strong>Deriving structured representations from natural language to aid scientific discovery: Case studies from reconstructing large pathway maps</strong><br />
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://suchee.bitbucket.io/">Sucheendra Kumar Palaniappan</a><br />
<i>The Systems Biology Institute, Tokyo, Japan</i></td></tr>
<tr><td><b>09:25 - 09:50 </b><br />
<strong>Talks selected from abstracts</strong><br /></td></tr>
</table> 

  
<table>
<tr><td><strong><font color="#e60000">Session 2: AI for network-based drug discovery and repositioning</font></strong></td></tr>
<tr><td><b>09:50 - 10:15 </b><br />
<strong>Pharma Databases Integration coupled with human curation for Drug Development/Repurposing</strong><br /> 
<a href="/Abstracts/Bionetvisa2019/Abstract_BioNetVisA2019_Junhyung_PARK.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="http://3bigs.com/en/about-us-eng/">Junhyung Park</a><br />
<i>3BIGS CO., LTD. Suwon, Republic of Korea</i></td></tr>
<tr><td><b>10:15 - 10:40 </b><br />
<strong>Network-based and data-driven drug discovery by machine learning</strong><br /> 
<a href="/Abstracts/Bionetvisa2019/Abstract_BioNetVisA2019_Yoshihiro_YAMANISHI.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="http://labo.bio.kyutech.ac.jp/~yamani/index.html">Yoshihiro Yamanishi</a><br />
<i>Kyushu Institute of Technology, Fukuoka, Japan</i></td></tr>
<tr><td><b>10:40 - 11:05</b><br /> 
<strong>Network-based knowledge for drug discovery: Application in pharmaceutical industry</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://www.linkedin.com/in/tatsuya-ando-4a14a3138/">Tatsuya Andou</a><br />
<i>Takeda Pharmaceuticals, Tokyo, Japan</i></td></tr>
</table>
 
<table> 
<tr><td><i>11:05 - 11:15 Break</i></td></tr>
</table>


<table>
<tr><td><strong><font color="#e60000">Session 3: Cancer evolution and medical applications of systems biology</font></strong></td></tr>
<tr><td><b>11:15 - 11:40 </b><br />
<strong>The Reconstruction of Cancer Phylogenies</strong><br /> 
<a href="/Abstracts/Bionetvisa2019/Abstract_BioNetVisA2019_David_WEDGE.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://www.bdi.ox.ac.uk/Team/david-wedge">David Wedge</a><br />
<i>Big Data Institute, University of Oxford, UK</i></td></tr>
<tr><td><b>11:40 - 12:05 </b><br />
<strong>Challenges to integrating systems biology and artificial intelligence in medical applications</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://www.researchgate.net/profile/Yoshiyuki_Asai">Yoshiyuki Asai</a><br />
<i>Yamaguchi University, Yamaguchi, Japan</i></td></tr>
<tr><td><b>12:05 - 12:30 </b><br />
<strong>Towards Data-Integrated Medicine</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://www.bsc.es/przulj-natasa">Natasa Przulj</a><br />
<i>Barcelona Supercomputing Center, Barcelona, Spain</i></td></tr>
<tr><td><b>12:30 - 13:00 </b><br />
<strong>Talks selected from abstracts</strong></td></tr>
</table>

<table>
<tr><td><i>13:00 - 14:00 Lunch</i></td></tr>
</table>

<table>
<tr><th><font color="#e60000">Session 4: Big data: integration, visualization interpretation</font></th></tr>
<tr><td><b>14:00 - 14:25</b><br />
<strong>Multi-omics and medical data integration and visualization</strong><br /> 
<a href="/Abstracts/Bionetvisa2019/Abstract_BioNetVisA2019_Reinhard_SCHNEIDER.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://wwwfr.uni.lu/lcsb/people/reinhard_schneider">Reinhard Schneider</a><br />
<i>Luxembourg Centre for Systems Biomedicine, Luxembourg</i></td></tr>
<tr><td><b>14:25 - 15:10</b><font color=color="#e60000"><i>Keynote</i></font></b><br />
<strong>Joint analysis of large single-cell dataset collections</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="http://pklab.med.harvard.edu/index.html">Peter Kharchenko</a><br />
<i>Harvard medical school, Boston, USA</i></td></tr>
<tr><td><b>15:10 - 15:35 </b><br />
<strong>Networks and trajectories in cancer</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://science.curie.fr/members/emmanuel-barillot-phd/">Emmanuel Barillot</a><br />
<i>Institut Curie, Paris, France</i></td></tr>
<tr><td><b>15:35 - 16:00 </b><br />
<strong>Talks selected from abstracts</strong></td></tr>
</table>

<table> 
<tr><td><i>16:00 - 16:10 Break</i></td></tr>
</table>

<table>
<tr><th><font color="#e60000">Session 5: Modelling of biological and disease networks</font></th></tr>
<tr><td><b>16:10 - 16:35</b><br />
<strong>Towards Predictive Biology: Applications and challenges of mechanistic modeling and machine learning</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="http://www.sbi.jp/members.htm">Ayako Yachie</a><br />
<i>The Systems Biology Institute, Tokyo, Japan</i></td></tr>
<tr><td><b>16:35 - 17:00 </b><br />
<strong>A universal network mechanism governing heterogenous ErbB signaling dynamics</strong><br /> 
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="http://www.protein.osaka-u.ac.jp/cell_systems/people.html">Hiroaki Imoto</a><br />
<i>Institute for Protein Research, Osaka University, Osaka, Japan</i></td></tr>
<tr><td><b>17:00 - 17:30</b><br />
<strong>Talks selected from abstracts</strong></td></tr>
<tr><td><b>17:30 - 18:00 </b><br />
<strong>Round table discussion and closing remarks</strong><br /></td></tr>
</table>
 
### Abstract submission instructions
<p>
We invite to submit your abstract for a <b>talk</b> in the topics described above.<br />
<font color="#e60000">The submission deadline is on August 1st 2019</font>
</p>
<p>Submitted abstracts will be reviewed by the scientific program committee and a notification of acceptance for a <b>talk</b> will be provided to the corresponding author by August 12th, 2018.
</p>
<p>Number of poster abstracts will be also selected for flash poster presentations (3 minutes of topic advertisement). 
</p>
<p><b>Abstract format:</b><br />
Title<br />
List of authors (the first author is the presenting author)<br />
Affiliations<br />
Abstract text in unstructured format (maximum 300 words)
</p>
<p align="justify"><b>Submission process</b><br />
The link below will take you to the EasyChair submission page. Once logged, click on the 'Submission' tab to start the submission process. Enter authors, title without HTML elements, abstract up to 300 words, keywords, and select up to <b><span style="text-decoration: underline;">three</span></b> relevant topic sessions.
</p>
<p align="justify">To assist us with creating the abstract booklet, please upload the same abstract as a word document in the section called "Upload Paper" using this <b><a href="/AbstractTemplate_BioNetVisA.docx"><span style="text-decoration: underline;"><font color="#e60000">TEMPLATE</font></span></a></b>
</p>
<p align="justify">When the form is adequately filled out, press the 'Submit' button. You will receive an email from the conference "EasyChair". This email is solely a notification that EasyChair for BioNetVisA2019 has received the abstract.
</p>
<b><a href="https://easychair.org/conferences/?conf=bionetvisa2019"><font size="5" color="#e60000"><center>Submit your abstract here</center></font></a></b>

### Registration

<p>The free-of-charge registration is obligatory for all participants (speakers, poster presenters, audience). <br /> 
<b><font color="#e60000">The registration deadline is on September 1st 2019</font></b><br />
<b><a href=""><font size="5" color="#e60000"><center>Register here</center></font></a></b>
</p>

### Organising committee
<p><a href="https://science.curie.fr/recherche/biologie-interactive-des-tumeurs-immunologie-environnement/c/">Emmanuel Barillot</a> (Institut Curie, France)<br />
<a href="http://www.sbi.jp/">Hioraki Kitano</a> (RIKEN Center for Integrative Medical Sciences, Japan)<br />
<a href="https://www.bsc.es/valencia-alfonso">Alfonso Valencia</a> (Spanish National Bioinformatics Institute, Madrid, Spain)<br/>
<a href="http://www.sbi.jp/members.htm">Samik Ghosh</a> (Systems Biology Institute, Tokyo, Japan)<br />
<a href="http://sysbio.curie.fr/">Inna Kuperstein</a> (Institut Curie, France)<br />
<a href="http://sysbio.curie.fr/">Luis Cristobal Monraz Gomez</a> (Institut Curie, France)<br />
<a href="https://reactome.org/">Robin Haw</a> (Ontario Institute for Cancer Research, Canada)<br />
<a href="http://www.ihes.fr/~zinovyev/">Andrei Zinovyev</a> (Institut Curie, France)<br />
<a href="https://www.prinsesmaximacentrum.com/">Patrick Kemmeren</a> (Princess Maxima Center for Pediatric Oncology, Utrecht, Netherlands)
</p>

### Venue 
<p> Okinawa Insitute of Science and Technology Graduate University, Okinawa, Japan<br />
1919-1 Tancha, Onna-son, Kunigami-gun, Okinawa, Japan 904-0495<br />
<a href="https://www.oist.jp/access-map">https://www.oist.jp/access-map</a><br />
Room: 2
</p>
      
<!--
### Scope
<p align="justify"><b>Development, curation and maintenance of biological network resources</b><br />
Data models from interaction to pathway and network, available tools for the integration of new discoveries into existing signalling diagrams. Discussion of curation models from map manager-oriented to community-based update using web-tools and forums.
</p>
<p align="justify"><b>Cell signalling knowledge representation paradigms</b><br />
From interaction to single pathways representation to comprehensive signalling networks: discussion of modes of biological processes representation; granularity, coverage, sources of information, annotation modes, cross-citations between different signalling networks resources, etc. Standard for graphical representation and data formats.
</p>
<p align="justify"><b>Visualization of data through networks</b><br />
Paradigms of visualization of different types of high-throughput data on a network map. Data integration with a map. Methods compatibility with different paradigms of signalling networks reconstruction in various databases and resources.
</p>
<p align="justify"><b>Analytical tools for network biology</b><br />
Tools for network analysis and manipulation, network modeling, network-based statistics. Studying network activation and inhibition, feedbacks and compensations, synthetic interactions.
</p>
<p align="justify"><b>Applications of networks in multi-omics data analysis in biomedical research and in clinics</b><br />
Studies on single-cell data, microbiome, metabolomics and other omics data analysis, disease
comorbidity, drug response prediction using network-based approaches.
</p>
### Sponsors
  <div>
      <a href="http://isac-net.org/" target="_blank"><img src="thumbnail_Logo_ISAC_sponsor.jpg" alt="INCa" style="float: left; width:30%; margin-right:10%; margin-bottom: 0.5em;"></a>
      </div>	-->


